src/main/java/com/sarahhand/fractals/model/MandelbrotConfig.java

```diff
@@ -85,7 +99,8 @@ public MandelbrotConfig(Double center, double zoom, int maxDwell, ColorPalette p
 	public MandelbrotConfig(@JsonProperty("x") double x, @JsonProperty("y") double y, 
 			@JsonProperty("zoom") double zoom, 
 			@JsonProperty("maxDwell") int maxDwell, 
-			@JsonProperty("palette") ColorPalette palette) {
+			@JsonProperty("palette") ColorPalette palette,
+			@JsonProperty("colorScheme") ColorScheme colorScheme) {
```

Comment:
I think we should remove colorScheme from this constructor until we figure out how to get Jackson parsing of the colorScheme working. Can you remove this argument for now?